### PR TITLE
Add coauthors to talk proposal.

### PR DESCRIPTION
**Pull request by tarsisazevedo**
This PR adds coauthors to proposal. 

Tasks:
- [x] translate to pt-br
- [x] add coauthors to proposal list in event's page
- [x] add label `coauthors` to related field in proposal's create page

If you see other tasks please, let me know!

### CODE REVIEW

File: deck/models.py [hunk 1]
```diff
@@ -243,6 +244,19 @@ def disapprove(self):
         self.is_approved = False
         self.save()
 
+    @property
```
Comment (mauricioabreu) [inline deck/models.py]: Why is this one a `property` and `get_authors_email` not?

File: deck/models.py [hunk 2]
```diff
@@ -243,6 +244,19 @@ def disapprove(self):
...
+    def coauthors_names(self):
+        coauthors = self.coauthors.all().values('username')
+        if not coauthors:
+            return _("None")
+        return ", ".join(map(lambda x: x['username'], coauthors))
+
+    def get_authors_email(self):
```
Comment (mauricioabreu) [inline deck/models.py]: `get_authors_emails` as a plural looks better since it returns a list of emails.

Comment (mauricioabreu) [inline deck/models.py]: Also you could avoid that loop using values_list.
Example:
```
# Documentation: https://docs.djangoproject.com/en/1.7/ref/models/querysets/#django.db.models.query.QuerySet.values_list
emails = self.coauthors.values_list('email', flat=True)
```

File: deck/tests/test_functional.py [hunk 3]
```diff
@@ -233,6 +245,21 @@ def test_notify_event_jury_and_proposal_author_on_new_proposal(self):
         self.assertIn(proposal.author.email, author_email.recipients())
         self.assertIn(settings.NO_REPLY_EMAIL, author_email.from_email)
 
+    def test_notify_proposal_author_and_coauthors_on_new_proposal(self):
+        event = Event.objects.create(**self.event_data)
+        coauthor = {"coauthors": 2}
```
Comment (mauricioabreu) [inline deck/tests/test_functional.py]: This `coauthor` variable is just used below. It could be set inside the update.

**Comment by mauricioabreu** [comment]
There are two similar migrations. Are they necessary? Can they be merged into one?

**Comment by mauricioabreu** [comment]
Good job! :-) 
One pattern that will probably emerge here is separate the Coauthor from the Author object. They will probably live in the same table but with different objects/behaviors.

```
                       / Authors
Users Table 
                       \ Coauthors
```

Not sure I am being clear here.

**Comment by mauricioabreu** [comment]
Much better @tarsisazevedo good job! :-)

**Comment by tarsisazevedo** [comment]
The label task can't be done because bootstrap css is overriding label by default.
screen shot 2016-01-20 at 12 29 01 am

If you don't have any other suggestion I think this PR can be merged! 

Thank for this awesome project. :)

**Comment by mauricioabreu** [comment]
:+1: 

Good job @tarsisazevedo 
One thing I would do here is "rebase" all those commits.

File: locale/pt_BR/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -1238,10 +1472,20 @@ msgid ""
...
+"people can submit proposals and engaje more participants!\n"
 "\n"
 "  Good luck!\n"
-msgstr "\nOi!\n\nEste e-mail é uma confirmação da criação do seu evento \"%(event_title)s\" no Speakerfight! Lembre-se de compartilhar o evento para que você ajudar a engajar mais participantes!\n\nBoa sorte!\n"
+msgstr ""
+"\n"
+"Oi!\n"
+"\n"
+"Este e-mail é uma confirmação da criação do seu evento \"%(event_title)s\" "
+"no Speakerfight! Lembre-se de compartilhar o evento para que você ajudar a "
```
Comment (mauricioabreu) [inline locale/pt_BR/LC_MESSAGES/django.po]: "você **possa** ajudar"

**Comment by luanfonceca** [comment]
@tarsisazevedo i will give my feedbacks for this

o/

**Comment by luanfonceca** [comment]
@tarsisazevedo Did you already seen how the Jury works?
See it in action: http://recordit.co/js8iJhVrbX

**Comment by mauricioabreu** [comment]
Hey @tarsisazevedo 

This PR covers lots of changes. Can you split the changes in multiple PRs?

**Comment by mauricioabreu** [comment]
About splitting this PR: that `makefile` can be a new pull request because it is not related to the feature.

**Comment by tarsisazevedo** [comment]
I will close this PR and open two more: one for coauthors feature (with proposed modifications) and one to Makefile fix.

**Pull request closed without merging**